draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1914,10 +1919,10 @@ SCTs and other `TransItem` structures.
 The appointed Expert should review the public specification to ensure that it is
 detailed enough to ensure implementation interoperability.
 
-## Extensions    {#extension_types}
+## Extension Types    {#extension_types}
 
-IANA is asked to establish a registry of SCT extensions, named "CT Extension
-Types for SCT", that initially consists of:
+IANA is asked to establish a registry of `ExtensionType` values, named "CT Log
+Artifact ExtensionTypes", that initially consists of:
```

Comment:
Actually, let's: 1) call it "CT Log Artifact Extensions", and 2) rename the "Value" column to "ExtensionType", and 3) rename the "Extension" column to "Status".